ruby-challenges.rb

```diff
@@ -19,16 +29,58 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum (nums_array)
+    nums_array.inject(0) {|sum, num| sum+=num}
```

Comment:
Nice!  Ruby also has a sum built-in method that does all the work automatically (and is faster to process)